setup.py

```diff
@@ -32,7 +32,7 @@
             "notes = lms.djangoapps.notes.views:NotesTab",
             "pdf_textbooks = lms.djangoapps.courseware.tabs:PDFTextbookTabs",
             "progress = lms.djangoapps.courseware.tabs:ProgressTab",
-            "static_tab = xmodule.tabs:StaticTab",
+            "static_tab = openedx.core.lib.course_tabs:StaticTab",
```

Comment:
I'm thinking this should be in lms.djangoapps like the majority of the other tabs.